src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/NormalizeModifiersOrOperatorsCodeCleanupProvider.vb

```diff
@@ -52,7 +52,13 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
             ' this order will be used when the rewriter re-order modifiers
             ' PERF: Using UShort instead of SyntaxKind as the element type so that the compiler can use array literal initialization
             Private Shared ReadOnly s_modifierKindsInOrder As SyntaxKind() =
```

Comment:
Should this code use the current value from the options rather then defaults?

Reply:
If this is indeed correct, I'll add test that the order is the same as in VisualBasicIdeCodeStyleOptions.Default.

Reply:
Added assert.